### PR TITLE
first changes, need to change locations

### PR DESCRIPTION
**Pull request by cranmer**

### CODE REVIEW

File: proposal.md [hunk 1]
```diff
@@ -48,13 +50,18 @@ enable the execution of Jupyter notebooks in highly configurable
...
 
 First and foremost, we need a compute-independent specification
-describing the paper repository layout and requirements, along with
-outputs; such a specification would permit introspection of the
-repository to support execution and composition on compatible
-platforms.
+describing the computational workflow including not only the 
+dependencies and environment needed to execute the code, but 
+also the meta-data needed to indicate the inputs, outputs, and configuration 
+parameters of the that describe the workflow at a semantic level.
```
Comment (betatim) [inline proposal.md]: missing word?

File: proposal.md [hunk 2]
```diff
@@ -82,27 +89,27 @@ tracking ([ORCID]).
 Sixth, the current "notebook" style of execution supported by both
 Jupyter and RMarkdown limits resuability, remixability, and
 composibility of code segments.  The rich Jupyter Notebook format
-specification also presents challenges for longevity of research
-artifacts. Before we "bake" notebooks into computational
+specification also presents challenges for long term preservation of research
```
Comment (betatim) [inline proposal.md]: For me "long term preservation" is a trigger word: trying to keep stuff running 20-50years from now with minimal effort. I think this is a seriously hard problem, like so hard I wouldn't want to try and solve it today. You can easily get tangled up in things that stop you from delivering a 80% solution tomorrow. "longevity" is more neutral for me. Probably depends on the reader.

Comment (betatim) [inline proposal.md]: The more often I read it, the more I realise I should have voiced my objections earlier. I don't think the notebook format itself creates new problems. Neither the specific format for storing a ipynb, nor the fact that you write things in a notebook format (mixing prose with code).

Will think about this and make a concrete suggestion tomorrow.

**Comment by betatim** [comment]
(house keeping: this needs a rebase before we can merge it. Not quite sure how you rebase as you made your changes in your master (always make a new branch :sweat:) maybe @lukasheinrich can help in your timezone)

I think most changes are fine, but need to read them again in the morning.

**Comment by cranmer** [comment]
I don't think I added anything about the notebook creating new problems. I just word-smithed that bullet point a bit. Anyways, new clean PR #73 where I took into account some of your comments already.

**Comment by betatim** [comment]
No you didn't. Like I said I should have raised this before as it was already there, you just attracted attention to it by reformulating it.

**Pull request closed without merging**